google-cloud-core-bom/pom.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.google.cloud</groupId>
+  <artifactId>google-cloud-core-bom</artifactId>
+  <version>1.31.1-SNAPSHOT</version><!-- {x-version-update:google-cloud-core:current} -->
+  <packaging>pom</packaging>
+  <parent>
+    <groupId>com.google.cloud</groupId>
+    <artifactId>google-cloud-shared-config</artifactId>
+    <version>0.1.0</version>
```

Comment:
should this version be synced with the rest of the repo?

Reply:
Fixed the version of the bom. The parent project is an external parent that contains the shared build/testing/release configuration across the split repositories.